examples/scripts/sparsecode_poisson_blkarr_pgm.py

```diff
@@ -13,14 +13,23 @@
 to solve the non-negative reconstruction problem with Poisson negative
 log likelihood loss
 
-  $$\mathrm{argmin}_{\mathbf{x}} \; \frac{1}{2} \left ( A \mathbf{x} -
-  \mathbf{y} \log\left( A \mathbf{x} \right) + \log(\mathbf{y}!) \right
-  ) + I(\mathbf{x} \geq 0)\;,$$
+  $$\mathrm{argmin}_{\mathbf{x}} \; \frac{1}{2} \left ( A(\mathbf{x}) -
+  \mathbf{y} \log\left( A(\mathbf{x}) \right) + \log(\mathbf{y}!) \right
```

Comment:
Is there a reason to write it as `A(\mathbf{x})` here? In most cases we've been writing it as `A \mathbf{x}`.

Reply:
`A` is not a linear operator. Have we been doing this even when `A` is not linear?

Reply:
Oh, sorry; in that case, please ignore this comment.